src/components/mainPage/UploadBox.jsx

```diff
@@ -26,6 +25,7 @@ const UploadBox = (props) => {
           Drag 'n' drop images here, or <b>click</b> to select files
         </p>
       )}
+      <p className="accepted-file-formats-text">Accepted file formats: {accept.join(", ")}</p>
```

Comment:
This doesn't generalise well, e.g. the issue Avi was having .jpeg files.

Reply:
I'm not sure what you mean, this is displaying the file types we accept. If we add .jpeg to the list then it will appear

Reply:
There are at least 6 extensions that correspond to jpeg files, so it may be better to just say JPEG and on hover display the valid file extensions. We eventually shouldn't be checking the extension at all, but getting the type from the header.

Reply:
Ah, you mean it doesn't scale well. Yeah fair enough

Reply:
Found out the library allows you to specify using the MIME type, so this should be addressed now